### ISSUE TITLE
Why Is the Wiki Unfinished?

### ISSUE THREAD

**Issue by TurtleShroom** [m00nl1ght-dev/GeologicalLandforms#5]
Good evening.

Is there any particular reason that the documentation for the Wiki is functionally non-existent? I am trying to make a custom Cavern that uses the Biomes Caverns Mod to render, and its dependence on your Mod calls many functions that are not unlisted.

Do you think you could expand on the documentation, that we may be able to best use this Mod's code?

Thank you for any assistance you may render.
-TURTLESHROOM

**Comment by m00nl1ght-dev** [comment]
The Wiki only contains documentation for features that are considered stable and therefore ready for use by other mods. All other features (which are **not** currently in the Wiki) are experimental and still under development. The only mods that use those features are mods that I'm personally involved in development wise, such as "Biomes! Caverns". Once an experimental feature has been sufficiently tested and optimized, it will eventually be marked as stable. Only at that point will it be added to the Wiki and is ready for use by other modders.

**Comment by TurtleShroom** [comment]
I'm not sure why you would do that, but I guess you know better about your own Mod than me. I tried to commission someone to add the Stalagmites to my custom Biome, but he never responded. For reasons beyond my own understanding, the Biomes Core Mod developers banned me from their entire community and never gave an explanation.

My hunch is that they banned me because I said how ridiculously dumb it was that NPC Factions could spawn on Ocean Biomes and expressed my opposition to walking on sea tiles. I recanted the latter but stood by the former, because frankly, it was and still is retarded. 


(E-Mail reply.)
------
Jesus Loves You and Died for You!

     On Sunday, May 28, 2023 at 05:09:36 AM EDT, m00nl1ght-dev ***@***.***> wrote:  
 
 


The Wiki only contains documentation for features that are considered stable and therefore ready for use by other mods. All other features (which are not currently in the Wiki) are experimental and still under development. The only mods that use those features are mods that I'm personally involved in development wise, such as "Biomes! Caverns". Once an experimental feature has been sufficiently tested and optimized, it will eventually be marked as stable. Only at that point will it be added to the Wiki and is ready for use by other modders.
